image-patching/stopgap-script/nuage_overcloud_full_patch.py

```diff
@@ -253,6 +254,22 @@ def copy_avrs_packages(image, avrs_baseurl, proxy_hostname = None, proxy_port =
     cmds_run(['rm -f nuage_avrs_packages'])
 
 
+#####
+# Installing Mellanox Packages
+#####
+
+def install_mellanox(image, workingDir):
+    # This is a temporary fix provided by Mellanox for os-net-config and will be removed in future releases
+    virt_copy('%s %s/os-net-config/os_net_config/* /usr/lib/python2.7/site-packages/os_net_config' % (image, workingDir))
```

Comment:
same as above

Reply:
Hi Moshe, I have updated the comment https://github.com/nuagenetworks/nuage-ospdirector/pull/54/files#diff-5678ac47e177f50d7f10526d27451edfR263. Please let me know how it looks. I am open to suggestions.